lxc/main.go

```diff
@@ -316,6 +316,22 @@ func (c *cmdGlobal) PreRun(cmd *cobra.Command, args []string) error {
 	// Set the user agent
 	c.conf.UserAgent = version.UserAgent
 
+	// Set the Candid API URL
+	cs, err := c.conf.GetContainerServer(c.conf.DefaultRemote)
+	if err != nil {
+		return err
+	}
+
+	server, _, err := cs.GetServer()
+	if err != nil {
+		return err
+	}
+
+	candidURL := server.Writable().Config["candid.api.url"]
+	if candidURL != nil {
+		c.conf.CandidURL = candidURL.(string)
+	}
```

Comment:
I don't see how this code can work. What if the user uses something other than the default remote, what if they do operations against multiple remotes?

Also in the case where the user is adding a new remote, they can't actually know the candid URL until after they've authenticated through candid, so I don't think this approach will work at all.